keycloak migration failed

**Describe the bug**
When running `wls-init-keycloak` container with fresh db it fails.

**To Reproduce**
- delete old containers: `wls-init-keycloak`, `wls-keycloak` and `wls-db-postgres-keycloak`
- run `wls-init-keycloak` container in docker-compose file (it will setup the required containers`)

**Expected behavior**
Migration was successful without errors

**Log**
<details>

<summary>Log</summary>

```log
[INFO ] 2024-08-30 07:08:15.031 [main] Action - Executing migration: AddRole Basisdaten_BUSINESSACTION_GetWahlbezirke
[INFO ] 2024-08-30 07:08:15.076 [main] Action - Executing migration: AssignRole Basisdaten_BUSINESSACTION_GetWahlbezirke to Group: allBasisdatenAuthorities
[INFO ] 2024-08-30 07:08:15.127 [main] Action - Executing migration: AddRole Basisdaten_READ_Wahltag
[ERROR] 2024-08-30 07:08:18.187 [main] KeycloakMigration - Error occurred while migrating: add authorities basisdaten wahlbezirke 
feign.FeignException$Conflict: [409 Conflict] during [POST] to [http://wls-keycloak:8080/auth/admin/realms/wls_realm/clients/e0a67a8e-c070-4122-8835-f454c3e52c6a/roles] [KeycloakClient#addClientRole(AddRole,UUID,String)]: [{"errorMessage":"Role with name Basisdaten_READ_Wahltag already exists"}]
        at feign.FeignException.clientErrorStatus(FeignException.java:234) ~[keycloakmigration.jar:?]
        at feign.FeignException.errorStatus(FeignException.java:203) ~[keycloakmigration.jar:?]
        at feign.FeignException.errorStatus(FeignException.java:194) ~[keycloakmigration.jar:?]
        at feign.FeignException.errorStatus(FeignException.java:171) ~[keycloakmigration.jar:?]
        at feign.codec.ErrorDecoder$Default.decode(ErrorDecoder.java:92) ~[keycloakmigration.jar:?]
        at feign.ResponseHandler.decodeError(ResponseHandler.java:136) ~[keycloakmigration.jar:?]
        at feign.ResponseHandler.handleResponse(ResponseHandler.java:70) ~[keycloakmigration.jar:?]
        at feign.SynchronousMethodHandler.executeAndDecode(SynchronousMethodHandler.java:114) ~[keycloakmigration.jar:?]
        at feign.SynchronousMethodHandler.invoke(SynchronousMethodHandler.java:70) ~[keycloakmigration.jar:?]
        at io.github.resilience4j.retry.Retry.lambda$decorateCheckedFunction$7bb28b04$1(Retry.java:187) ~[keycloakmigration.jar:?]
        at io.github.resilience4j.feign.DecoratorInvocationHandler.invoke(DecoratorInvocationHandler.java:95) ~[keycloakmigration.jar:?]
        at com.sun.proxy.$Proxy29.addClientRole(Unknown Source) ~[?:?]
        at de.klg71.keycloakmigration.changeControl.actions.role.AddRoleAction.execute(AddRoleAction.kt:48) ~[keycloakmigration.jar:?]
        at de.klg71.keycloakmigration.changeControl.actions.Action.executeIt(Action.kt:37) ~[keycloakmigration.jar:?]
        at de.klg71.keycloakmigration.changeControl.KeycloakMigration.doChange(KeycloakMigration.kt:45) [keycloakmigration.jar:?]
        at de.klg71.keycloakmigration.changeControl.KeycloakMigration.execute$keycloakmigration(KeycloakMigration.kt:31) [keycloakmigration.jar:?]
        at de.klg71.keycloakmigration.MainKt$migrate$1$1.invoke(Main.kt:74) [keycloakmigration.jar:?]
        at de.klg71.keycloakmigration.MainKt$migrate$1$1.invoke(Main.kt:66) [keycloakmigration.jar:?]
        at org.koin.core.context.GlobalContext.startKoin(GlobalContext.kt:65) [keycloakmigration.jar:?]
        at org.koin.core.context.DefaultContextExtKt.startKoin(DefaultContextExt.kt:31) [keycloakmigration.jar:?]
        at de.klg71.keycloakmigration.MainKt.migrate(Main.kt:66) [keycloakmigration.jar:?]
        at de.klg71.keycloakmigration.MainKt$main$1.invoke(Main.kt:22) [keycloakmigration.jar:?]
        at de.klg71.keycloakmigration.MainKt$main$1.invoke(Main.kt:20) [keycloakmigration.jar:?]
        at com.xenomachina.argparser.SystemExitExceptionKt.mainBody(SystemExitException.kt:74) [keycloakmigration.jar:?]
        at com.xenomachina.argparser.SystemExitExceptionKt.mainBody$default(SystemExitException.kt:72) [keycloakmigration.jar:?]
        at de.klg71.keycloakmigration.MainKt.main(Main.kt:20) [keycloakmigration.jar:?]
[ERROR] 2024-08-30 07:08:18.194 [main] KeycloakMigration - Rolling back changes
[INFO ] 2024-08-30 07:08:18.194 [main] Action - Undo migration: AssignRole Basisdaten_BUSINESSACTION_GetWahlbezirke to Group: allBasisdatenAuthorities
[INFO ] 2024-08-30 07:08:18.244 [main] Action - Undo migration: AddRole Basisdaten_BUSINESSACTION_GetWahlbezirke
[ERROR] 2024-08-30 07:08:18.315 [main] KeycloakMigration - Migrations were unsuccessful:
feign.FeignException$Conflict: [409 Conflict] during [POST] to [http://wls-keycloak:8080/auth/admin/realms/wls_realm/clients/e0a67a8e-c070-4122-8835-f454c3e52c6a/roles] [KeycloakClient#addClientRole(AddRole,UUID,String)]: [{"errorMessage":"Role with name Basisdaten_READ_Wahltag already exists"}]
        at feign.FeignException.clientErrorStatus(FeignException.java:234) ~[keycloakmigration.jar:?]
        at feign.FeignException.errorStatus(FeignException.java:203) ~[keycloakmigration.jar:?]
        at feign.FeignException.errorStatus(FeignException.java:194) ~[keycloakmigration.jar:?]
        at feign.FeignException.errorStatus(FeignException.java:171) ~[keycloakmigration.jar:?]
Exception in thread "main" feign.FeignException$Conflict: [409 Conflict] during [POST] to [http://wls-keycloak:8080/auth/admin/realms/wls_realm/clients/e0a67a8e-c070-4122-8835-f454c3e52c6a/roles] [KeycloakClient#addClientRole(AddRole,UUID,String)]: [{"errorMessage":"Role with name Basisdaten_READ_Wahltag already exists"}]
        at feign.FeignException.clientErrorStatus(FeignException.java:234)
        at feign.FeignException.errorStatus(FeignException.java:203)
        at feign.FeignException.errorStatus(FeignException.java:194)
        at feign.FeignException.errorStatus(FeignException.java:171)
        at feign.codec.ErrorDecoder$Default.decode(ErrorDecoder.java:92)
        at feign.ResponseHandler.decodeError(ResponseHandler.java:136)
        at feign.ResponseHandler.handleResponse(ResponseHandler.java:70)
        at feign.SynchronousMethodHandler.executeAndDecode(SynchronousMethodHandler.java:114)
        at feign.SynchronousMethodHandler.invoke(SynchronousMethodHandler.java:70)
        at io.github.resilience4j.retry.Retry.lambda$decorateCheckedFunction$7bb28b04$1(Retry.java:187)
        at io.github.resilience4j.feign.DecoratorInvocationHandler.invoke(DecoratorInvocationHandler.java:95)
        at com.sun.proxy.$Proxy29.addClientRole(Unknown Source)
        at de.klg71.keycloakmigration.changeControl.actions.role.AddRoleAction.execute(AddRoleAction.kt:48)
        at de.klg71.keycloakmigration.changeControl.actions.Action.executeIt(Action.kt:37)
        at de.klg71.keycloakmigration.changeControl.KeycloakMigration.doChange(KeycloakMigration.kt:45)
        at de.klg71.keycloakmigration.changeControl.KeycloakMigration.execute$keycloakmigration(KeycloakMigration.kt:31)
        at de.klg71.keycloakmigration.MainKt$migrate$1$1.invoke(Main.kt:74)
        at de.klg71.keycloakmigration.MainKt$migrate$1$1.invoke(Main.kt:66)
        at org.koin.core.context.GlobalContext.startKoin(GlobalContext.kt:65)
        at org.koin.core.context.DefaultContextExtKt.startKoin(DefaultContextExt.kt:31)
        at de.klg71.keycloakmigration.MainKt.migrate(Main.kt:66)
        at de.klg71.keycloakmigration.MainKt$main$1.invoke(Main.kt:22)
        at de.klg71.keycloakmigration.MainKt$main$1.invoke(Main.kt:20)
        at com.xenomachina.argparser.SystemExitExceptionKt.mainBody(SystemExitException.kt:74)
        at com.xenomachina.argparser.SystemExitExceptionKt.mainBody$default(SystemExitException.kt:72)
        at de.klg71.keycloakmigration.MainKt.main(Main.kt:20)
        at feign.codec.ErrorDecoder$Default.decode(ErrorDecoder.java:92) ~[keycloakmigration.jar:?]
        at feign.ResponseHandler.decodeError(ResponseHandler.java:136) ~[keycloakmigration.jar:?]
        at feign.ResponseHandler.handleResponse(ResponseHandler.java:70) ~[keycloakmigration.jar:?]
        at feign.SynchronousMethodHandler.executeAndDecode(SynchronousMethodHandler.java:114) ~[keycloakmigration.jar:?]
        at feign.SynchronousMethodHandler.invoke(SynchronousMethodHandler.java:70) ~[keycloakmigration.jar:?]
        at io.github.resilience4j.retry.Retry.lambda$decorateCheckedFunction$7bb28b04$1(Retry.java:187) ~[keycloakmigration.jar:?]
        at io.github.resilience4j.feign.DecoratorInvocationHandler.invoke(DecoratorInvocationHandler.java:95) ~[keycloakmigration.jar:?]
        at com.sun.proxy.$Proxy29.addClientRole(Unknown Source) ~[?:?]
        at de.klg71.keycloakmigration.changeControl.actions.role.AddRoleAction.execute(AddRoleAction.kt:48) ~[keycloakmigration.jar:?]
        at de.klg71.keycloakmigration.changeControl.actions.Action.executeIt(Action.kt:37) ~[keycloakmigration.jar:?]
        at de.klg71.keycloakmigration.changeControl.KeycloakMigration.doChange(KeycloakMigration.kt:45) ~[keycloakmigration.jar:?]
        at de.klg71.keycloakmigration.changeControl.KeycloakMigration.execute$keycloakmigration(KeycloakMigration.kt:31) [keycloakmigration.jar:?]
        at de.klg71.keycloakmigration.MainKt$migrate$1$1.invoke(Main.kt:74) [keycloakmigration.jar:?]
        at de.klg71.keycloakmigration.MainKt$migrate$1$1.invoke(Main.kt:66) [keycloakmigration.jar:?]
        at org.koin.core.context.GlobalContext.startKoin(GlobalContext.kt:65) [keycloakmigration.jar:?]
        at org.koin.core.context.DefaultContextExtKt.startKoin(DefaultContextExt.kt:31) [keycloakmigration.jar:?]
        at de.klg71.keycloakmigration.MainKt.migrate(Main.kt:66) [keycloakmigration.jar:?]
        at de.klg71.keycloakmigration.MainKt$main$1.invoke(Main.kt:22) [keycloakmigration.jar:?]
        at de.klg71.keycloakmigration.MainKt$main$1.invoke(Main.kt:20) [keycloakmigration.jar:?]
        at com.xenomachina.argparser.SystemExitExceptionKt.mainBody(SystemExitException.kt:74) [keycloakmigration.jar:?]
        at com.xenomachina.argparser.SystemExitExceptionKt.mainBody$default(SystemExitException.kt:72) [keycloakmigration.jar:?]
        at de.klg71.keycloakmigration.MainKt.main(Main.kt:20) [keycloakmigration.jar:?]
```

</details>

**Additional context**
The role `Basisdaten_READ_Wahltag` is part of `add-authorities-basisdaten-wahltage.yml` and `add-authorities-basisdaten-wahlbezirke.yml`

--- a/stack/keycloak/migration/add-authorities-basisdaten-wahlbezirke.yml
+++ b/stack/keycloak/migration/add-authorities-basisdaten-wahlbezirke.yml
@@ -12,24 +12,6 @@ changes:
       clientId: ${SSO_CLIENT_ID}
 
   - addRole:
-      name: Basisdaten_READ_Wahltag
-      clientRole: true
-      clientId: ${SSO_CLIENT_ID}
-  - assignRoleToGroup:
-      group: allBasisdatenAuthorities
-      role: Basisdaten_READ_Wahltag
-      clientId: ${SSO_CLIENT_ID}
-
-  - addRole:
-      name: Basisdaten_READ_Wahl
-      clientRole: true
-      clientId: ${SSO_CLIENT_ID}
-  - assignRoleToGroup:
-      group: allBasisdatenAuthorities
-      role: Basisdaten_READ_Wahl
-      clientId: ${SSO_CLIENT_ID}
-
-  - addRole:
       name: Basisdaten_READ_Wahlbezirk
       clientRole: true
       clientId: ${SSO_CLIENT_ID}

